lib/saluki-components/src/relays/otlp/mod.rs

```diff
@@ -144,9 +145,10 @@ impl Relay for OtlpRelay {
                     break;
                 },
                 Some(otlp_payload) = payload_rx.recv() => {
+                    let output_name = otlp_payload.signal_type.to_string();
```

Comment:
Can you make `OtlpSignalType` have an `as_str` method so we can return a `&'static str` and avoid an allocation every time we dispatch?